src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/linelength/Example6.java

```diff
@@ -14,13 +15,13 @@
 package com.puppycrawl.tools.checkstyle.checks.sizes.linelength;
 
 public class Example6 {
-  void testMethod() {
+  void testMethod() {// ok, the violation is suppressed
```

Comment:
Please use this wording of trailing comment 
https://checkstyle.org/filters/suppresswithnearbycommentfilter.html#Example2-code